### ISSUE TITLE
Successive undo-ing has become sluggish 2

### ISSUE THREAD

**Issue by mrkybe** [aseprite/aseprite#3388]
[This issue seems to have regressed?](https://github.com/aseprite/aseprite/issues/1695)

I'm noticing a significant difference in undo-speed based on the size of the color palette/color wheel.

### Aseprite and System version

* Aseprite version: 1.2.35 x64 Steam
* System: Windows 10

**Comment by dacap** [comment]
Hi @mrkybe, could you please try unchecking the *Edit > Preferences > Experimental > Use shaders for color selectors* option to see if the performance issue disappears?

**Comment by mrkybe** [comment]
> Hi @mrkybe, could you please try unchecking the Edit > Preferences > Experimental > Use shaders for color selectors option to see if the performance issue disappears?

Yep that seems to work.  Specifically, it seems that if I make like ~15 marks, with the shader option enabled, I can start undoing them and it'll start to lag behind how fast I'm pressing Ctrl+Z.  Whereas without the option enabled, I can undo them as fast as I press the key without any delay.
